google/cloud/compute/samples/disk_samples.cc

```diff
@@ -87,7 +87,8 @@ void CreateEmptyDisk(std::vector<std::string> const& argv) {
   }
   //! [END compute_disk_create_empty_disk]
   (argv.at(0), argv.at(1), argv.at(2), argv.at(3),
-   SplitLabelsString(argv.at(4)));
+   ((argv.size() == 5) ? SplitLabelsString(argv.at(4))
+                       : std::vector<std::pair<std::string, std::string>>{}));
```

Comment:
Aside: You might also consider that `std::vector<std::pair<std::string, std::string>>` smells like `std::[multi]map<std::string, std::string>`, and particularly seeing as the latter appears to match `Disk.labels`.